<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Курсовая работа</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::WeatherClear"/>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
	background-color: rgb(210, 215, 220);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="Textlabel">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Работа с данными:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane{
    border: 1px;
	background: rgb(175, 200, 220);
}

QTabBar::tab{
	background: rgb(80, 155, 255);
}

QTabBar::tab:selected{
	background: rgb(175, 200, 220);
}</string>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="Vvodtab">
       <attribute name="title">
        <string>Ввод</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="vvodgridLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="electfilelabel">
            <property name="text">
             <string>Ввод данных об оформлениях электронных книг из файла:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="electchislabel">
            <property name="text">
             <string>Введите количество оформлений для электронных книг:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="izdanchisspinBox">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="electfilepushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 155, 255);</string>
            </property>
            <property name="text">
             <string>Ввод</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="electfilelineEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="izdanchislabel">
            <property name="text">
             <string>Введите количество оформлений для изданных книг:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="electchisspinBox">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="izdanfilelabel">
            <property name="text">
             <string>Ввод данных об оформлениях изданных книг из файла:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="izdanfilelineEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="izdanfilepushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 155, 255);</string>
            </property>
            <property name="text">
             <string>Ввод</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="electformpushButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(80, 155, 255);</string>
          </property>
          <property name="text">
           <string>Ввод данных об оформлениях электронных книг из формы</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="electformvvodgridLayout">
          <item row="1" column="2">
           <widget class="QSpinBox" name="electformspinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLineEdit" name="electformlineEdit4">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="electformlineEdit3">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="electformlineEdit2">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="electformvvodlabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Для оформления 1:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="electformlineEdit1">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QPushButton" name="electformvvodpushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 155, 255);</string>
            </property>
            <property name="text">
             <string>Ввод</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="electformlabel1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Печать:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="electformlabel2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Количество страниц:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="electformlabel3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Обложка:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="electformlabel4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Иллюстрации:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="electformlabel5">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>Язык:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="izdanformpushButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(80, 155, 255);</string>
          </property>
          <property name="text">
           <string>Ввод данных об оформлениях изданных книг из формы</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="6">
           <widget class="QPushButton" name="izdanformvvodpushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 155, 255);</string>
            </property>
            <property name="text">
             <string>Ввод</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="izdanformlineEdit1">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="izdanformlineEdit2">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="izdanformspinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="izdanformlineEdit3">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="izdanformvvodlabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Для оформления 1:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLineEdit" name="izdanformlineEdit4">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="izdanformlabel1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Печать:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="izdanformlabel2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Количество страниц:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="izdanformlabel3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Обложка:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="izdanformlabel4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Иллюстрации:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="izdanformlabel5">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>Язык:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Sorttab">
       <attribute name="title">
        <string>Сортировка</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="sortgridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="electsortlabel">
            <property name="text">
             <string>Выберите поле для сортировки оформлений электронных книг:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="electsortpushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 155, 255);</string>
            </property>
            <property name="text">
             <string>Отсортировать</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="electsortcomboBox">
            <item>
             <property name="text">
              <string>Название</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Кол-во страниц</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Обложка</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Иллюстрации</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Язык</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="izdanlabel">
            <property name="text">
             <string>Выберите поле для сортировки оформлений изданных книг:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="izdanpushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 155, 255);</string>
            </property>
            <property name="text">
             <string>Отсортировать</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="izdansortcomboBox">
            <item>
             <property name="text">
              <string>Название</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Кол-во страниц</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Обложка</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Иллюстрации</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Язык</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Poisktab">
       <attribute name="title">
        <string>Поиск</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="poiskgridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="poisklabel">
            <property name="text">
             <string>Введите значение которое хотите найти:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="poiskpushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 155, 255);</string>
            </property>
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="poisklineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="poisktableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Vivodtab">
       <attribute name="title">
        <string>Вывод</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="udalhorizontalLayout">
          <item>
           <widget class="QLabel" name="udallabel">
            <property name="text">
             <string>Для удаления элемента введите его номер:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="udalspinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="udalpushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 155, 255);</string>
            </property>
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="infohorizontalLayout">
          <item>
           <widget class="QLabel" name="infolabel">
            <property name="text">
             <string>Для вывода сведений о книге введите номер оформления:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="infospinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="infopushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 155, 255);</string>
            </property>
            <property name="text">
             <string>Вывод</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Functtab">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <attribute name="title">
        <string>Обработка</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="maxstranpushButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(80, 155, 255);</string>
          </property>
          <property name="text">
           <string>Найти оформление с наибольшим кол-вом страниц</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="maxstranlabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="illustpushButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(80, 155, 255);</string>
          </property>
          <property name="text">
           <string>Найти каких иллюстраций больше</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="illustlabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="raznyazpushButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(80, 155, 255);</string>
          </property>
          <property name="text">
           <string>Найти кол-во оформлений на разных языках</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="raznyazlabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
